usr/share/rear/output/USB/Linux-i386/300_create_extlinux.sh

```diff
@@ -84,6 +84,12 @@ BEGIN {
     fi
 }
 
+# set nocasematch option
+shopt -s nocasematch
```

Comment:
When a shell option is set
it must be also unset when it is no longer needed
cf. the get_bash_flags_and_options_commands() and
apply_bash_flags_and_options_commands() functions
in usr/sbin/rear
or
the code that needs to run with a shell option is set
must run in a subshell, cf. how `shopt -s nocasematch` is used in
usr/share/rear/format/USB/default/200_check_usb_layout.sh

The comment `set nocasematch option` is useless
because to does not tell WHY that shell option must be set,
cf. "Code must be easy to understand (answer the WHY)" in
https://github.com/rear/rear/wiki/Coding-Style

Perhaps I am somehow blind but I do not see
why the nocasematch option must be set here.

As far as I understand it the intent is that
USB_BOOTLOADER could be 'syslinux' or 'extlinux' ignoring the case
but in general config variables have fixed syntax values
where in particular case matters (as usual in Unix).

Reply:
I was thinking about unsetting but when looking at other code places where it is used I found some where it was also not unset so I decided to do it the same there.
Yes, the intent was to check the value of `USB_BOOTLOADER` case insensitive but thats not a must. As a simple solution I will remove the option.

Reply:
Yes, the simpler the better!

I find 'nocasematch' only in usr/share/rear/format/USB/default/200_check_usb_layout.sh
```
localhost:~/rear.github.master # find usr/sbin/rear usr/share/rear/ -type f | xargs grep 'nocasematch'
usr/share/rear/format/USB/default/200_check_usb_layout.sh:    shopt -s nocasematch
```
and there it is set within a subshell so it has no effect outside of the subshell like
```
# shopt -p | grep nocasematch ; foo=$( shopt -s nocasematch ; shopt -p | grep nocasematch ) ; echo $foo ; shopt -p | grep nocasematch
shopt -u nocasematch
shopt -s nocasematch
shopt -u nocasematch
```